src/contacts/util/SerializationUtil.java

```diff
@@ -0,0 +1,22 @@
+package contacts.util;
+
+import java.io.*;
+
+public class SerializationUtil {
```

Comment:
what is this class for?

Reply:
This class for serialization and deserialization objects